app/src/main/kotlin/com/simplemobiletools/calculator/helpers/GetCurrencyTask.kt

```diff
@@ -0,0 +1,54 @@
+package com.simplemobiletools.calculator.helpers
+
+import android.os.AsyncTask
+import android.util.Log
+import com.beust.klaxon.JsonObject
+import com.beust.klaxon.Parser
+import com.simpletools.calculator.commons.helpers.Calculator
+import com.simpletools.calculator.commons.helpers.Formatter
+import java.io.BufferedInputStream
+import java.io.BufferedReader
+import java.io.InputStreamReader
+import java.net.HttpURLConnection
+import java.net.URL
+
+class GetCurrencyTask(val from: String, val to: String, moneyActivity: Calculator, moneyCalculator: MoneyCalculatorImpl) : AsyncTask<Void, Void, String>() {
+
+    var mCalc: Calculator = moneyActivity
+    var moneyCalculator: MoneyCalculatorImpl = moneyCalculator
+
+    override fun doInBackground(vararg params: Void?): String? {
+        val url = URL("https://v3.exchangerate-api.com/pair/6d45cb0dc607b808bc2e7758/$from/$to")
```

Comment:
Nice API. Better than the last one. Allows for lightweight requests! 👍

Reply:
Agreed !